modules/setup/setup.js

```diff
@@ -0,0 +1,222 @@
+                    this.checkPermission();
+              } else {
+                    console.log(Helper.outputError('Docker not detected on the system please install docker....'));
+              }
+        });
+    }
+
+    startNodes() {
+        child_process.exec(`docker images ${IMAGE_TAG} --format '{{.Repository}}:{{.Tag}}:{{.Size}}'`,
+        (error, __stdout, __stderr) => {
+            if (__stdout === `${IMAGE_TAG}:3GB\n`) {
+
+                child_process.exec(`docker ps -a -q  --filter ancestor=${IMAGE_TAG} --format '{{.Image}}:{{.Names}}'`,
+                (error, __stdout, __stderr) => {
+                    if (__stdout.includes(`${IMAGE_TAG}:${CONTAINER_NAME}\n`)) {
+                          console.log(Helper.outputInfo('Nodes are already running....'));
+                    } else {
+                          this.runContainer();
+                    }
+                });
+            } else {
+                  console.log(Helper.outputError(`No inbuilt nodes found on system. Run 'tezster setup' comamnd for build the nodes.`));
+              }
+        });
+    }
+
+    stopNodes() {
+        child_process.exec(`docker ps -a -q --format '{{.Image}}'`,(error, __stdout, __stderr) => {
+            if (__stdout.includes(`${IMAGE_TAG}\n`)) {
+                console.log(`stopping the nodes....`);
+                const container = docker.getContainer(CONTAINER_NAME);
+                docker.listContainers(function(err, containers) {
+                    container.stop(); 
+                    container.remove({force: true});
+                    console.log(Helper.outputInfo(`Nodes have been stopped. Run 'tezster start-nodes' to restart.`));
+                });
+            } else {
+               console.log(Helper.outputError('No Nodes are running....'));  
+            } 
+        });
+    }
+
+    getLogFiles() {
+        console.log(Helper.outputInfo(`getting log file....please wait for some time`));
+        const container = docker.getContainer(CONTAINER_NAME);
+
+        const fs = require('fs');
+        container.getArchive( { path: `/usr/local/bin/tezster-logs` }, (error, stream) => {
+            let writeStream = fs.createWriteStream('/tmp/tezster-logs.tar.gz', { encoding: 'utf8' });
+
+            if (error) {
+                console.error('get archive error', error);
+                writeStream.end();
+                writeStream.close();
+            }
+
+            stream.on('data', (data) => {
+                writeStream.write(data);
+            });
+        
+            stream.on('end', () => {
+                writeStream.end();
+                writeStream.close();
+                console.log(Helper.output(`log files stored as archieve format at: \n     '/tmp/tezster-logs.tar.gz'`))
+            });
+        });
+    }
+
+    checkPermission(){
+        child_process.exec(`stat -c '%a %n' ${__dirname}/config.json`,(error, __stdout, __stderr) => {
```

Comment:
remove all __ from variable names.

Reply:
__dirname is the directory position of the config.json. From any other variables name I have removed double underscore, using single underscore for private variables.